pc_helper_online.py

```diff
@@ -47,6 +51,23 @@ def main():
                     apply_publiccloud_pint_image(settings)
                     if "PUBLIC_CLOUD_IMAGE_ID" not in settings:
                         log.error(f"Failed to get PUBLIC_CLOUD_IMAGE_ID from {data}")
+                if "SLES4SAP_PINT_QUERY" in settings and "incidents" in data:
+                    for flavor in data["incidents"]["FLAVOR"]:
+                        if (
+                            "pint_name_regexp"
```

Comment:
@asmorodskyi  @pdostal this is only partially implemented but relevant clue about how next PR will handle metadata. As reference here a metadata example:

```
 product: SAP-15-SP5
   2   │ settings:
   3   │   DISTRI: sle
   4   │   VERSION: '15-SP5'
   5 + │   SLES4SAP_PINT_QUERY: 'https://susepubliccloudinfo.suse.com/v1'
   6   │ aggregate:
...
  18   │ incidents:
  19   │   FLAVOR:
...
  65   │     Azure-SAP-BYOS-Incidents-saptune:
  66   │       aggregate_job: false
  67   │       archs:
  68   │         - x86_64
  69   │       issues: *test_issues_list
  70   │       required_issues: *required_list
  71   │       packages: *whitelist_saptune
  72 + │       pint_name_regexp: '.*sap-15-sp5-byos'

```

Reply:
Why `SLES4SAP_PINT_QUERY` and not `PUBLIC_CLOUD_PINT_QUERY`?

Reply:
In order to call `apply_sles4sap_pint_image` in place of `apply_publiccloud_pint_image`